metadata-ingestion/src/datahub/sql_parsing/sqlglot_lineage.py

```diff
@@ -1091,6 +1090,28 @@ def sqlglot_lineage(
         return SqlParsingResult.make_from_error(e)
 
 
+sqlglot_lineage_cached = functools.lru_cache(maxsize=SQL_PARSE_RESULT_CACHE_SIZE)(
```

Comment:
```suggestion
_sqlglot_lineage_cached = functools.lru_cache(maxsize=SQL_PARSE_RESULT_CACHE_SIZE)(
```

same for the other method - we want to make it clear what the public-facing interface is